src/main/java/io/microprofile/config/Config.java

```diff
@@ -67,21 +63,7 @@
      */
     Boolean getBoolean(String propertyName);
```

Comment:
It can be achieved via the getPropertyValue but it is much nicer to directly call getBoolean. Agree?